full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -186,12 +190,12 @@ pub enum JsonCommandRequest {
     get_gift_code {
         gift_code_b58: String,
     },
-    get_mc_protocol_transaction {
-        transaction_log_id: String,
-    },
-    get_mc_protocol_txo {
-        txo_id: String,
-    },
+    // get_mc_protocol_transaction {
```

Comment:
delete commented code

Reply:
Why is this being deleted?

Reply:
It isn't, re-enabling it now! had it commented out while working on stuff.